.github/workflows/run-tests.yml

```
@@ -487,12 +488,24 @@ jobs:
           annotate_only: true
           skip_annotations: true
 
+      - name: Get job ID from workflow run
+        id: get_job_id
+        run: |
+          set -e          
+          job_id=$(curl -L \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{secrets.GITHUB_TOKEN}}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/"${GITHUB_REPOSITORY}"/actions/runs/"${GITHUB_RUN_ID}"/jobs \
+            | jq "first(.jobs[] | select(.name | endswith(\"$job_name\")) | .id)")
+          echo "JOB_ID=${job_id}" >> $GITHUB_OUTPUT
```

it's silly but this is what it's complaining about:
```suggestion
          echo "JOB_ID=${job_id}" >> "$GITHUB_OUTPUT"
```